src/Panel.vue

```diff
@@ -287,6 +287,9 @@
           this.$refs.retriever.fetch()
         }
       })
+      const panelHeader = this.$slots.header ? this.$refs.headerWrapper.innerHTML : this.headerContent;
+      const id = jQuery(panelHeader).wrap('<div></div>').parent().find(':header').text();
+      this.$refs.cardContainer.setAttribute('id', id.trim().replace(/\s+/g, '-').toLowerCase());
```

Comment:
If we cannot find a header text (i.e. `panelHeaderText` is empty), we should not set an `id` attribute.